react/javascript/demo-extension-sdk/README.md

```diff
@@ -0,0 +1,72 @@
+# Demo Extension SDK
+
+This example usage of the Looker Extension SDK using javascript. It does NOT demonstrate how to use the core SDK in an extension.
+
+## Getting Started for Development
+
+1. Clone the Looker extension examples repo.
+
+   ```
+   # cd ~/ Optional, your user directory is usually a good place to git clone to.
+   git clone git@github.com:looker-open-source/extension-examples.git
+   ```
+
+2. Navigate (`cd`) to the example directory on your system
+
+   ```
+   cd extension-examples/react/javascript/demo-extension-sdk
+   ```
+
+3. Install the dependencies with [Yarn](https://yarnpkg.com/).
+
+   ```
+   yarn install
+   ```
+
+4. Start the development server
+
+   ```
+   yarn develop
+   ```
+
+   The extension is now running and serving the JavaScript locally at http://localhost:8080/bundle.js.
+
+5. Log in to Looker and create a new project.
+
+   This is found under **Develop** => **Manage LookML Projects** => **New LookML Project**.
+
+   Select "Blank Project" as your "Starting Point". This will create a new project with no files.
+
+   1. The extension folder has a `manifest.lkml` file.
+
+   Either drag & upload this file into your Looker project, or create a `manifest.lkml` with the same content. Change the `id`, `label`, or `url` as needed.
+
+6. Create a `model` LookML file in your project. The name doesn't matter but the convention is to name it the same as the project— in this case, demo-extension-sdk.
+
+- Add a connection in this model.
+- [Configure the model you created](https://docs.looker.com/data-modeling/getting-started/create-projects#configuring_a_model) so that it has access to the selected connection.
+  We do this because Looker permissions data access via models— In order to grant / limit access to an extension, it must be associated with a model.
```

Comment:
```suggestion
  We do this because Looker permits data access via models— In order to grant / limit access to an extension, it must be associated with a model.
```

Don't feel strongly about this change.